### ISSUE TITLE
log scaling?

### ISSUE THREAD

**Issue by bbolker** [rhiever/baby-name-explorer#5]
It would be nice to have an option for log scaling on the y axis (especially in conjunction with the multiple-names-at-once enhancement)

**Comment by rhiever** [comment]
Seems like a reasonable feature. But as evidenced by my other issues sitting there untouched for a couple years, I'm not actively developing this project.

**Comment by bbolker** [comment]
Fair enough.   (The project came up on my Twitter feed this morning.)  I'm very inexperienced with D3 but I might have a look if I have some time.

**Comment by rhiever** [comment]
PRs are absolutely welcome!
